mentor_classifier/mentor_classifier/arch/transformers/embeddings.py

```diff
@@ -0,0 +1,20 @@
+#
+# This software is Copyright ©️ 2020 The University of Southern California. All Rights Reserved.
+# Permission to use, copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and subject to the full license file found in the root of this software deliverable. Permission to make commercial use of this software may be obtained by contacting:  USC Stevens Center for Innovation University of Southern California 1150 S. Olive Street, Suite 2300, Los Angeles, CA 90115, USA Email: accounting@stevens.usc.edu
+#
+# The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+#
+from mentor_classifier.sentence_transformer import find_or_load_sentence_transformer
+from sentence_transformer import SentenceTransformer
+
+
+class TransformerEmbeddings:
+    def __init__(self, shared_root):
+        self.transformer: SentenceTransformer = find_or_load_sentence_transformer(
+            shared_root + "/../sentence-transformer"
```

Comment:
use path for joins...also not sure why we need the backstep in path?

```python
find_or_load_sentence_transformer(os.path.join(shared_root, "sentence-transformer")
```

...also, will save some headache if we can add a file extension to the the `sentence-transformer` pretrained model, e.g. `sentence-transformer.bin`...because in the actual app deployment, it will need to be committed to git and distinguished from other files as using git-lfs

Reply:
The backstep is there so sentence_transformer is on the same level as Word2Vec. I'm not sure why Word2Vec was set up that way though.